Prace_domowe/Praca_domowa7/Jamrozy_Bartosz/pd7.ipynb

```diff
@@ -0,0 +1,926 @@
+{
```

Comment:
<p>w detekcji outlierów raczej definiuje się tylko jeden próg czyli w tym wypadku -26.5. Wynika to, z tego że próg jest logarytmem prawdopodobieństwa do klasy inliers. Dlatego im logarytm jest niższy tym większa szansa na to, że dana obserwacja jest outlierem. Niemniej jednak ustawienie dwóch granic nie jest błędem - jest tylko nadmiarowe.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/mini-pw/2021L-WUM/pull/291/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>